src/Response.php

```diff
@@ -93,8 +93,16 @@ public function __construct(
         $version = '1.1',
         $reason = null
     ) {
+        if (filter_var($status, FILTER_VALIDATE_INT) === false) {
+            throw new \InvalidArgumentException('Status code must be an integer(ish) value.');
+        }
+
         $this->statusCode = (int) $status;
 
+        if (!isset(self::$phrases[$this->statusCode])) {
```

Comment:
@Tobion or whoever else wants.. 

I added an extra check here... WDYT?

Reply:
This must be reverted as you can have custom status codes that are not listed.

Reply:
done.